dev/io.openliberty.data.internal_fat_jpa/test-applications/DataJPATestApp/src/test/jakarta/data/jpa/web/DataJPATestServlet.java

```diff
@@ -2928,6 +2932,132 @@ public void testOneToOne() {
         drivers.deleteByFullNameEndsWith(" TestOneToOne");
     }
 
+    @Test
+    public void testPersistentFieldNamesAndDelimiters() {
```

Comment:
Add a Javadoc comment to this test with a sentence stating what it covers. That information, along with the test name, will eventually be copied into the Feature Test Summary to get our test approval.
```suggestion
    /**
     * Tests entity attribute names from embeddables and MappedSuperclass that
     * can have delimiters. Includes tests for name collisions with attributes from an
     * embeddable or superinteface.
     */
    @Test
    public void testPersistentFieldNamesAndDelimiters() {
```